base/cvd/cuttlefish/host/commands/cvd/unittests/selector/instance_database_helper.cpp

```diff
@@ -55,11 +55,17 @@ bool Touch(const std::string& full_path) {
 }  // namespace
 
 CvdInstanceDatabaseTest::CvdInstanceDatabaseTest()
-    : error_{.error_code = ErrorCode::kOk, .msg = ""} {
+    : error_{.error_code = ErrorCode::kOk, .msg = ""},
+      db_backing_path_("/tmp/cvd_test_db_XXXXXX"),
+      db_backing_fd_(mkstemp(db_backing_path_.data())),
```

Comment:
Should something make sure `mkstemp` didn't fail?

Reply:
Done